_posts/10-07-2022

```diff
@@ -0,0 +1,34 @@
+---
+title:  "Tableau Tabcmd 2 - Release for general use (Tableau Online functionality)"
```

Comment:
We have been standardizing on tabcmd (both in 1.0 and 2.0), although that seems to the command only. If we want to standardize on something else, this would be a good time to do so. I've seen support refer to it as "TabCMD" and then there's "Tabcmd" noted above. Any preference?

Reply:
tabcmd it is